.eslintrc.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
     "arrow-spacing": [
       "error", { "before": true, "after": true }
     ],
-    "no-console": 0,
+    "no-console": 2,
```

Comment:
(nit) I feel this could be made a bit more explicit, like use `"no-console": "error"` instead of a magic number

Reply:
Done